### ISSUE TITLE
binascii.Error: Incorrect padding when trying to convert to pdf

### ISSUE THREAD

**Issue by lainglo** [jupyter/nbconvert#579]
from the command:

`jupyter nbconvert --to pdf notebook.ipynb`

Here is the trace:

```
Traceback (most recent call last):
  File "/usr/bin/jupyter-nbconvert", line 11, in <module>
    sys.exit(main())
  File "/usr/lib/python3.6/site-packages/jupyter_core/application.py", line 267, in launch_instance
    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)
  File "/usr/lib/python3.6/site-packages/traitlets/config/application.py", line 658, in launch_instance
    app.start()
  File "/usr/lib/python3.6/site-packages/nbconvert/nbconvertapp.py", line 305, in start
    self.convert_notebooks()
  File "/usr/lib/python3.6/site-packages/nbconvert/nbconvertapp.py", line 473, in convert_notebooks
    self.convert_single_notebook(notebook_filename)
  File "/usr/lib/python3.6/site-packages/nbconvert/nbconvertapp.py", line 444, in convert_single_notebook
    output, resources = self.export_single_notebook(notebook_filename, resources, input_buffer=input_buffer)
  File "/usr/lib/python3.6/site-packages/nbconvert/nbconvertapp.py", line 373, in export_single_notebook
    output, resources = self.exporter.from_filename(notebook_filename, resources=resources)
  File "/usr/lib/python3.6/site-packages/nbconvert/exporters/exporter.py", line 171, in from_filename
    return self.from_file(f, resources=resources, **kw)
  File "/usr/lib/python3.6/site-packages/nbconvert/exporters/exporter.py", line 189, in from_file
    return self.from_notebook_node(nbformat.read(file_stream, as_version=4), resources=resources, **kw)
  File "/usr/lib/python3.6/site-packages/nbconvert/exporters/pdf.py", line 162, in from_notebook_node
    nb, resources=resources, **kw
  File "/usr/lib/python3.6/site-packages/nbconvert/exporters/latex.py", line 82, in from_notebook_node
    return super(LatexExporter, self).from_notebook_node(nb, resources, **kw)
  File "/usr/lib/python3.6/site-packages/nbconvert/exporters/templateexporter.py", line 236, in from_notebook_node
    nb_copy, resources = super(TemplateExporter, self).from_notebook_node(nb, resources, **kw)
  File "/usr/lib/python3.6/site-packages/nbconvert/exporters/exporter.py", line 131, in from_notebook_node
    nb_copy, resources = self._preprocess(nb_copy, resources)
  File "/usr/lib/python3.6/site-packages/nbconvert/exporters/exporter.py", line 308, in _preprocess
    nbc, resc = preprocessor(nbc, resc)
  File "/usr/lib/python3.6/site-packages/nbconvert/preprocessors/base.py", line 47, in __call__
    return self.preprocess(nb,resources)
  File "/usr/lib/python3.6/site-packages/nbconvert/preprocessors/base.py", line 70, in preprocess
    nb.cells[index], resources = self.preprocess_cell(cell, resources, index)
  File "/usr/lib/python3.6/site-packages/nbconvert/preprocessors/extractoutput.py", line 80, in preprocess_cell
    data = a2b_base64(data)
binascii.Error: Incorrect padding
```

Hopefully just a case of missing configs/settings on my system?

**Comment by takluyver** [comment]
Can you share a copy of the notebook (in a gist)? I think that indicates that an output is invalid base64. We could at least fail with a more helpful error, though, or even warn and continue without that output.

**Comment by lainglo** [comment]
Thanks. I made a gist [here](https://gist.github.com/lainglo/e7e4e79a66b380ace46df93b9a4a0699).

I have been working on the notebook since posting, and when I tried to convert just now I get the error:
`nbconvert.utils.pandoc.PandocMissing: Pandoc wasn't found.`

Not sure what happened to the earlier error.

I tried to install pandoc but seems like all the arch repo's are down or something. I will have to try again tomorrow.
